### PR TITLE
Rename analytics.skroutz.gr/analytics.min.js script name

### PR DESCRIPTION
**Pull request by alexunder193**

### CODE REVIEW

**Review by MrGoumX — APPROVED** [review]
+1 LGTM! 🚀

**Review by mrwhizzy — APPROVED** [review]
+1 LGTM!

File: source/localizable/analytics/settings.html.md.erb [hunk 1]
```diff
@@ -42,7 +42,7 @@ queue from `<%= "#{flavor}_analytics" %>()` to whatever name you choose.
 (function(a,b,c,d,e,f,g){a['<%=flavor.camelize%>AnalyticsObject']=e;a[e]= a[e] || function(){
   (a[e].q = a[e].q || []).push(arguments);};f=b.createElement(c);f.async=true;
   f.src=d;g=b.getElementsByTagName(c)[0];g.parentNode.insertBefore(f,g);
-})(window,document,'script','https://analytics.skroutz.gr/analytics.min.js','custom_analytics_name');
```
Comment (harrisred) [inline source/localizable/analytics/settings.html.md.erb:45]: [minor] We should have used the `analytics_domain` config here as well:
```
})(window,document,'script','https://<%=settings.analytics_domain%>/skroutza.min.js','custom_analytics_name');
```

Comment (harrisred) [reply source/localizable/analytics/settings.html.md.erb:45]: fixed: https://github.com/skroutz/developer.skroutz.gr/pull/158/commits/1c369e7ce2b869504fe21994441567563d724316

**Review by harrisred — APPROVED** [review]
LGTM 👍

**Pull request merged**